src/object/cli_shard.c

```diff
@@ -306,6 +354,8 @@ dc_rw_cb_csum_verify(const struct rw_cb_args *rw_args)
 			}
 			break;
 		}
+
+		rw_args_store_csum(rw_args, iod_csum);
```

Comment:
Just curious, do we have the option to not even verify csum on rebuild "client" side, i.e. only inserting the csum gotten from fetch or enumeration?

Reply:
Currently no, but it's something to think about.